src/components/applications/DevApplicationDetails.js

```diff
@@ -0,0 +1,106 @@
+import {ListElementDetails} from "../commons/layouts/ListElementDetails";
+import {CardContent, Divider, Grid, Typography, Button} from "@material-ui/core";
+import {FileViewerWrapper} from "./FileViewerWrapper";
+import {processAPI} from "../../utils/apis/ProcessAPI";
+import { useState, useEffect } from 'react'
+import { AssignTasks } from "../tasks/AssignTasks";
+import { tasksApi } from "../../utils/apis/tasksAPI";
+import { useDevPassword } from "../../utils/hooks/useDevPassword";
+
+export const DevApplicationDetails = ({devApplication}) => {
+
+    const application = devApplication?.application
+
+    const { getEncoded } = useDevPassword()
+    const devPassword = getEncoded()
+
+    const [openAssignTasks, setOpenAssignTasks] = useState(false)
+    const [alreadyAssignedTasks, setAlreadyAssignedTasks] = useState([])
+    const [organizationTasks, setOrganizationTasks] = useState([])
+    const [reload, setReload] = useState(false)
+
+    const getSeekerFiles = () => {
+        return application.seekerFiles
+            .map((file, idx) => (
+                <Grid item key={`${idx}`} xs={12} md={6}>
```

Comment:
Tu nie wystarczyłoby samo `key={idx}`?

Reply:
Pewnie tak. Na swoją obronę powiem, że to jest copy paste z ApplicationDetails, a wtedy jeszcze myślałem, że key musi być stringiem XD